tripal_chado/src/ChadoBuddy/ChadoBuddyPluginBase.php

```diff
@@ -92,20 +97,20 @@ public function description(): string {
    *
    * @throws Drupal\tripal_chado\ChadoBuddy\Exceptions\ChadoBuddyException
    *   For invalid table name.
-   **/
+   */
   protected function getTableColumns(array $chado_tables, string $filter = 'all') {
```

Comment:
Let's try to consisently type hint all params and return value.

```suggestion
  protected function getTableColumns(array $chado_tables, string $filter = 'all'): array {
```

Reply:
It looks like getTableCache() below is missing this too but I can't comment there as no changes were made.